cranelift/codegen/src/isa/zkasm/lower.isle

```diff
@@ -102,29 +97,28 @@
   (zk_divu x y))
 
 (rule (lower (has_type $I32 (udiv x y)))
-  (zk_divu_32 x y))
+  (zk_divu x y))
 
 
 (rule (lower (has_type $I64 (sdiv x y)))
   (zk_div x y))
 
 (rule (lower (has_type $I32 (sdiv x y)))
-  (zk_div_32 x y))
-
+  (zk_div x y))
 
 ;;;; Rules for `rem` ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
 (rule (lower (has_type $I64 (srem x y)))
   (zk_rem x y))
 
 (rule (lower (has_type $I32 (srem x y)))
-  (zk_rem_32 x y))
+  (zk_rem x y))
 
 (rule (lower (has_type $I64 (urem x y)))
   (zk_remu x y))
 
 (rule (lower (has_type $I32 (urem x y)))
-  (zk_remu_32 x y))
+  (zk_remu x y))
```

Comment:
Drop the `has_type` predicate and remove one of the rules.

Reply:
Done